src/box/relay.cc

```diff
@@ -516,6 +545,21 @@ relay_initial_join(struct iostream *io, uint64_t sync, struct vclock *vclock,
 
 	/* Send read view to the replica. */
 	engine_join_xc(&ctx, &relay->stream);
+	say_info("read-view sent.");
+}
+
+void
+relay_initial_join(struct iostream *io, uint64_t sync,
+		   struct vclock *vclock, uint32_t replica_version_id,
+		   const struct vclock *cursor_vclock, int64_t cursor_lsn)
+{
+	if (unlikely(cursor_vclock != NULL && vclock_is_set(cursor_vclock))) {
```

Comment:
Nitpicking, but lets drop `unlikely()`. That is certainly one of the coldest paths in the codebase where an `if` optimization gives no impact.

Reply:
Dropped